qemu/tests/x86_cpu_flags.py

```diff
@@ -36,7 +38,17 @@ def run(test, params, env):
     error_context.context("Try to log into guest", logging.info)
     session = vm.wait_for_login()
     if params["os_type"] == "linux":
+        check_guest_cmd = params.get("check_guest_cmd", "")
         check_cpu_flags(params, flags, test, session)
+        if check_guest_cmd:
+            check_items = params.get("check_items", "")
+            check_no_items = params.get("check_no_items", "")
+            if check_items and not cpu_check_cmd_output(
+                    check_guest_cmd, check_items, session):
+                test.fail("'%s' can't be found inside guest" % check_items)
+            if check_no_items and cpu_check_cmd_output(
```

Comment:
Will it be used by another test case? It can be implemented by executing "check_guest_cmd | grep -v check_no_items" in guest without function cpu_check_cmd_output.

Reply:
> Will it be used by another test case? It can be implemented by executing "check_guest_cmd | grep -v check_no_items" in guest without function cpu_check_cmd_output.

Yes, it will be used by another test case. update and thanks.